examples/timer/src/main.rs

```diff
@@ -4,16 +4,20 @@ use yew::services::{ConsoleService, Task, TimeoutService};
 use yew::{html, Callback, Component, ComponentLink, Html, ShouldRender};
 
 pub enum Msg {
+    StartClock,
     StartTimeout,
     StartInterval,
     Cancel,
     Done,
     Tick,
+    Tock,
 }
 
 pub struct Model {
     link: ComponentLink<Self>,
     job: Option<Box<dyn Task>>,
+    clock_job: Option<Box<dyn Task>>,
```

Comment:
Let's use `Option<IntervalTask>` instead of a trait object here (Like `_standalone` already does).